### ISSUE TITLE
lack of antialiasing

### ISSUE THREAD

**Issue by Gollam** [AlexxIT/go2rtc#646]
I have tried both the picture-entity card, and your custom webrtc as well, the result is the same. The card shows the camera stream correctly, but it is very very pixelated, I can barely read out the timestamp. I tried this both in Firefox and Chrome. I have 4MP Hikvision cameras, I use RTSP protocol. Is there some setting I haven't enabled? I guess it is not intended behavior. 
![image](https://github.com/AlexxIT/go2rtc/assets/765342/704fccd0-34ca-47b7-9d23-1c8d05724a9a)

**Comment by AlexxIT** [comment]
What resolution of your camera in pixels? What OS you using?

**Comment by Gollam** [comment]
@AlexxIT this is a 4MP stream, the os is windows 11.

**Comment by AlexxIT** [comment]
Pixels please

**Comment by Gollam** [comment]
Oh, ok. The 4MP one is 2560 × 1440, and I have a 5MP, which is 2560 × 1920.

**Comment by AlexxIT** [comment]
Both streams has same problem?

**Comment by Gollam** [comment]
Yes, the distortion looks exactly the same on both streams. Should I try lowering the resolution or anything else, for debugging purposes?

**Comment by AlexxIT** [comment]
I can confirm this problem on Windows Edge. Don't have problem on Mac. Don't know how to fix it in near future.
